dashboard/src/components/AppView/AccessURLTable/AccessURLItem/IURLItem.tsx

```diff
@@ -0,0 +1,6 @@
+export interface IURLItem {
```

Comment:
shouldn't we put this in the shared dir? or maybe types.ts, although IMO that file is huge and we should break types out somehow